src/index.js

```diff
@@ -854,22 +880,38 @@ class MiniCssExtractPlugin {
         }
 
         generate() {
-          const { chunk, runtimeRequirements } = this;
+          const { chunkGraph, chunk, runtimeRequirements } = this;
           const {
             runtimeTemplate,
-            outputOptions: { crossOriginLoading },
+            outputOptions: { chunkLoadingGlobal, crossOriginLoading },
           } = /** @type {Compilation} */ (this.compilation);
           const chunkMap = getCssChunkObject(
             /** @type {Chunk} */ (chunk),
             /** @type {Compilation} */ (this.compilation)
           );
+          const { globalObject } = runtimeTemplate;
+          const { linkPreload, linkPrefetch } =
+            JsonpChunkLoadingRuntimeModule.getCompilationHooks(compilation);
```

Comment:
Please put these hooks here, i.e. just copy/paste, because this plugin can work not only with Jsonp, also with ESM modules